fecgov/fec-cms
# Feature/5679 sass minifying

## Summary

- Resolves #5679 

Modernizing our sass minifying process to include the latest and greatest features (e.g. container queries) and to move away from packages with declining support (and more likely to become security vulnerabilities)

### Required reviewers

2? (feel free to remove yourself or others—I tagged more than needed)

## Impacted areas of the application

Every time we build our sass files:
- `npm run build`
- `npm run build-sass`
- `npm run build-production`
- `npm run watch-css`

## Screenshots

none

## Related PRs

none

## How to test
- Pull the branch
- `npm i`
- Any of the `npm run` scripts referenced here should not throw errors
- `./manage.py runserver`
- Pages should load normally and be styled as expected (check at least he homepage, a data page, and a Wagtail page)
